iOS/MMIWG2S/Views/ARFaceViewerUI.swift

```diff
@@ -66,28 +78,75 @@ class ARFaceViewerUI: UIView {
         redButton.centerYAnchor.constraint(equalTo: toggleView.centerYAnchor).isActive = true
         self.redButton = redButton
     }
-    
-    @objc private func shareButtonTapped() {
-        guard let imageToShare = delegate?.generateShareImage() else { return }
-        
-        // Use this to upload to the database. This will likely move :)
-//        let scaledDownImage = imageToShare.compress(to: 1000)
-        
-        let activityController = UIActivityViewController(activityItems: [imageToShare], applicationActivities: nil)
-        activityController.completionWithItemsHandler = { [weak self] (activityType, completed:Bool, returnedItems:[Any]?, error: Error?) in
-            self?.delegate?.resumeSessionOnceCaptured()
+
+    func setupShareBackButtons() {
+        if let shareIcon = UIImage(named: "share-icon"),
```

Comment:
this works! I'd also suggest just having a singular `setupButton` function that would take a `name: String`, `title: String` and an `isTailing: Bool`. Then there wouldn't need to be any copy paste code.

Reply:
Of course, you are right :) I did this closer to midnight and thought, oh it's fine, but Callum will have something to say about this :)

Reply:
Haha yeah either way works! If it runs it runs 😄

Reply:
Thanks for keeping me honest :)